components/select/Select.js

```diff
@@ -276,6 +320,7 @@ class Select extends PureComponent {
           styles={this.getStyles()}
           {...restProps}
         />
+        {error ? this.renderValidationMessage() : this.renderHelpText()}
```

Comment:
Why not just call it `renderError`?

Reply:
I called it `renderValidationMessage` because it would be expandable with other message types, like you can see in the design guidelines.

![schermafdruk 2018-09-17 09 41 25](https://user-images.githubusercontent.com/5336831/45611069-72c4ea80-ba5e-11e8-8de9-c35bc1f241aa.png)

But, the other message types are not implemented yet. So, I can change the name to `renderError` for now if you want?